packages/react-scripts/config/webpack.config.ssr.js

```diff
@@ -189,22 +187,32 @@ module.exports = function (webpackEnv) {
     ].filter(Boolean);
     if (preProcessor) {
       loaders.push(
-        {
-          loader: require.resolve('resolve-url-loader'),
-          options: {
-            sourceMap: isEnvProduction ? shouldUseSourceMap : isEnvDevelopment,
-            root: paths.appSrc,
+        ...[
+          {
+            loader: require.resolve('resolve-url-loader'),
+            options: {
+              sourceMap: isEnvProduction
+                ? shouldUseSourceMap
+                : isEnvDevelopment,
+              root: paths.appSrc,
+            },
           },
-        },
-        {
-          loader: require.resolve(preProcessor),
-          options: {
-            ...preProcessorOptions,
-            ...{
-              sourceMap: true,
+          // shouldUseCacheLoader && {
```

Comment:
The comment from the main file is missing here, can it be added?

Reply:
Yeah, add comments too.